app/use_cases/enrich_content.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+
+# Enrich response content (= questionnaire answers) with scores
+class EnrichContent < ActiveInteraction::Base
+  EnrichMissingDataError = Class.new(StandardError)
+
+  hash :content, strip: false # strip: false means allow all keys
+  hash :questionnaire, strip: false
+
+  # Enriches the questionnaire outcomes with scores
+  #
+  # @param content [Hash] the response values as a hash (keys are strings)
+  # @param questionnaire [Hash] the questionnaire definition hash (keys can be symbols)
+  def execute
+    @questionnaire = questionnaire # work with instance variables instead of activeinteraction methods
```

Comment:
why?

Reply:
Out of laziness because otherwise my IDE freaks out. Also there was some weird edge case if the value was nil, .blank? would return false or something.

I'll put it back and see if it causes the specs to fail.

Reply:
Seems to still be okay, so, fixed.